profile/ffsn.py

```diff
@@ -62,12 +73,14 @@ def profile_ffsn(n_trials):
 
     # plot results
     fig, ax = plt.subplots()
-    util.comparison_plot(proc_time, proc_time_std, n_std, ax)
+    comparison_plot(proc_time, proc_time_std, n_std, ax)
     ax.set_xlabel("Number of FS coefficients")
+    ax.set_xticks(np.array(N_FS_vals) - 1)
+    ax.set_yticks([1e-4, 1e-3, 1e-2, 1e-1, 1e0, 1e1])
     fig.tight_layout()
+    fig.savefig(os.path.join(fig_path, "profile_ffsn_2d.png"))
 
-    fname = pathlib.Path(__file__).resolve().parent / "profile_ffsn_2d.png"
-    fig.savefig(fname, dpi=300)
+    plt.show()
```

Comment:
`fig.show()`

Reply:
Perhaps it's my matplotlib settings, but `fig.show()` doesn't generate a pop-up window with the figure, while `plt.show()` does.